tools/ui_integration/setup_ui_integration.py

```diff
@@ -0,0 +1,160 @@
+from data_repo_client import Configuration, ApiClient, ProfilesApi, DatasetsApi, SnapshotsApi, JobsApi
+import argparse
+import subprocess
+import json
+import os
+import time
+import uuid
+
+DATASETS_TO_UPLOAD = [
+    {
+        "name": "V2F_GWAS_Summary_Statistics",
+        "tables": [
+            "ancestry_specific_meta_analysis",
+            "feature_consequence",
+            "dataset_specific_analysis",
+            "frequency_analysis",
+            "trans_ethnic_meta_analysis",
+            "transcript_consequence",
+            "variant",
+        ],
+        "format": "json",
+        "stewards": ["JadeStewards-dev@dev.test.firecloud.org"],
+        "custodians": [],
+        "snapshot_creators": []
+    },
+    {
+        "name": "NonStewardDataset",
+        "tables": [],
+        "format": "json",
+        "stewards": [],
+        "custodians": [],
+        "snapshot_creators": ["JadeStewards-dev@dev.test.firecloud.org"]
+    }
+]
+
+
+class Clients:
+    def __init__(self, host):
+        config = Configuration()
+        config.host = host
+        token_output = subprocess.run(['gcloud', 'auth', 'print-access-token'], capture_output=True)
```

Comment:
Is there any information about what user (if any) is expected to set up the data?

Reply:
Anyone with access can run it.

Reply:
Oh, I know anyone can run it but then you have to make sure that the test user has access too

Reply:
The test user is a part of the dev jade stewards, which is given the appropriate access

Reply:
derp, you're totally right